sdk/src/Services/DynamoDBv2/Custom/DataModel/Configs.cs

```diff
@@ -140,6 +140,11 @@ public DynamoDBContextConfig()
         /// Service calls made via <see cref="AmazonDynamoDBClient"/> will always return 
         /// <see cref="DateTime"/> attributes in UTC.</remarks>
         public bool? RetrieveDateTimeInUtc { get; set; }
+
+        /// <summary>
+        /// Property indicating the name of the attribute used for derived types conversion.
```

Comment:
Expand on this documentation. I assume this is the attribute that stores the metadata used by this library to figure out the type to instantiate.

This attribute is missing listing the default attribute.